src/features/instance/applications/editor/components/TextEditorView/index.tsx

```diff
@@ -41,25 +80,35 @@ export function TextEditorView() {
 		<div className="h-full">
 			<div className="flex items-center justify-between py-1 border-b border-gray-700">
 				<span className="p-2">{selectedFolderFile.filePath ? crumbPath : 'Select a file'}</span>
-				<Button
-					variant="positiveOutline"
-					className="w-32 rounded-full"
-					onClick={() => {
-						onSaveFile({
-							file: selectedFolderFile.filePath.split('/').slice(2).join('/'),
-							payload: updateFileContent,
-							project: selectedFolderFile.projectName,
-						});
-					}}
-					disabled={
-						!selectedFolderFile.filePath ||
-						(selectedFolderFile.filePath && updateFileContent == selectedFolderFile.content) ||
-						isSavingFile
-					}
-				>
-					<Save />
-					<span className="ms-1">Save</span>
-				</Button>
+				<div>
+					<Button
+						variant="positiveOutline"
+						className="w-32 rounded-full"
+						onClick={() => {
+							onSaveFile({
+								file: selectedFolderFile.filePath.split('/').slice(2).join('/'),
+								payload: updateFileContent,
+								project: selectedFolderFile.projectName,
+							});
+						}}
+						disabled={
+							!selectedFolderFile.filePath ||
+							(selectedFolderFile.filePath && updateFileContent == selectedFolderFile.content) ||
+							isSavingFile
+						}
+					>
+						<Save />
+						<span className="ms-1">Save</span>
+					</Button>
+					<Button
```

Comment:
Can we include a tooltip giving some more information about the impact of restarts? I think it is probably important to inform users that the full restart on the config page has about 100x the impact of this restart. Maybe a tooltip with verbiage:

> Performs a rolling restart of all the threads running this service to ensure all application changes are applied. This should not result in any downtime, but may incur some slower performance during the restart process.

In contrast, it would be nice to add a tooltip to the config page restart stating:

> This will perform a full restart of the entire Harper service and will incur downtime while the restart is taking place. Please use the restart on the application tab for restarting without downtime.